app/src/pages/playground/PromptComboBox.tsx

```diff
@@ -90,22 +81,3 @@ function PromptComboBoxComponent({
     </ComboBox>
   );
 }
-
-function PromptComboBoxLoader(props: PromptComboBoxProps) {
-  const [queryReference, loadQuery, disposeQuery] =
-    useQueryLoader<PromptComboBoxQuery>(promptComboBoxQuery);
-
-  useEffect(() => {
-    // TODO(apowell): Paginate and filter in query
-    loadQuery({ first: 100 });
-    return () => disposeQuery();
-  }, [disposeQuery, loadQuery]);
```

Comment:
lazy load is simpler, useQueryLoader doesn't really have any benefits if we aren't hoisting the query higher up anyway

Reply:
another option is to make it have a refetchable fragment. Not urgent